app/views/shared/fields/_corporate_name.html.erb

```diff
@@ -15,14 +15,14 @@
 
 		htmlOpen %= 'Corporate Name'.pluralize(fieldData.count)
 		concat htmlOpen.html_safe
+%>
 
-		fieldData.each do |datum|
-			corporateName = datum
-			concat "<li>#{corporateName}</li>".html_safe
-		end
-
-		concat htmlClose.html_safe
+	  <ul class="unstyled glyphicon-list glyphicon-list-tag">
+	    <% fieldData.each do |topic| %>
+	      <li><%= link_to topic, facet_uri(topic, 'subject_topic_sim') %></li>
```

Comment:
this goes for each of the additions of this block in the PR. Are these each still in the general `subject_topic_sim` facet or are they in their own?